assopy/tests/test_assopy_user.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth import get_user_model
+
+import pytest
+
+from assopy.models import AssopyUser
+
+
+@pytest.mark.parametrize(
+    "first_name,last_name,expected", [
+        ("John", "Doe", ("John", "Doe")),
+        ("John Doe", "Doe", ("John", "Doe")),
+        # let's get some utf8 going
+        ("Miroslav", "Šedivý", ("Miroslav", "Šedivý")),
+        ("Miroslav Šedivý", "Šedivý", ("Miroslav", "Šedivý")),
```

Comment:
Great way of test parametrization! I don't think we should have test cases based on an actual conference attendees though, worst case scenario, if he was to invoke GDPR, it's another thing to worry about.

Reply:
Fair enough :))

As I was thinking about unicode [Miro's talk](https://de.pycon.org/program/pydata-zna8wn-your-name-is-invalid-miroslav-sedivy/) came to mind  and I just couldn't help myself.